### DISCUSSION TITLE
jotai docs for more language ?

### DISCUSSION THREAD

**Discussion by SquirrelJimmy** [pmndrs/jotai#555] [General]
as title said, translate jotai doc for more language!

**Comment by dai-shi** [comment]
If we have enough volunteers!
@Thisen is working on renewing docs system, so maybe after it's done?

**Reply by SquirrelJimmy** [reply]
> If we have enough volunteers!
> @Thisen is working on renewing docs system, so maybe after it's done?

That's perfect! Someone is so lazy for using if a lib  has not a docs for his mother language!
